src/env-vars-check.ts

```diff
@@ -174,6 +174,7 @@ if (PROVIDERS.github) {
 if (ENV.AUTH_SMS_PROVIDER) {
   if (ENV.AUTH_SMS_PROVIDER === 'twilio') {
     [
+      'APP_NAME',
```

Comment:
this introduces a breaking change: SMS authentication won't work in upgrading to the new version without setting a value to this new `APP_NAME` environment variable.
I would make sure what's implemented does not introduce any regression

Reply:
`APP_NAME` env variable is needed in my opinion for serving many purposes like emails, sms, webauthn relying party name and maybe many more. I wonder why there wasn't  one till one 🤔 I assumed that such variable easily can be available in Nhost. 

Rather introducing breaking change we can fallback to some default string, but I'm not sure if it's a good idea, imagine:

`123456 is your verification code for hasura-auth`

Let me know if there's something to adjust here.